conanfile.py

```diff
@@ -17,8 +17,13 @@ def requirements(self):
         if self.options.build_unittest:
             self.requires.add('doctest/2.3.4@bincrafters/stable')
 
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.install()
+
     def package(self):
         self.copy("*LICENSE", dst="licenses")
-        self.copy("*.h", src=".")
-        self.copy("*.hpp", src=".")
+        self.copy("*.hpp", src="include")
+        self.copy("*.hpp", src="lib")
```

Comment:
This includes the files:
* lib/cmake/FunctionalPlus/FunctionalPlusConfig.cmake
* lib/cmake/FunctionalPlus/FunctionalPlusConfigVersion.cmake
* lib/cmake/FunctionalPlus/FunctionalPlusTargets.cmake

Reply:
Hi @Dobiasd, it is the cmake files, but as pointed out by @danimtb, there is no need for it. So I have removed it in the lastest commit.